app/lib/sms-games/controllers/SGCompetitiveStoryController.js

```diff
@@ -102,7 +103,7 @@ SGCompetitiveStoryController.prototype.createGame = function(request, response)
       gameDoc.betas[idx] = {};
       gameDoc.betas[idx].invite_accepted = false;
       gameDoc.betas[idx].phone = phone;
-      gameDoc.betas[idx].name = request.body['beta_first_name_' + i] || null;
+      gameDoc.betas[idx].name = request.body['beta_first_name_' + i] || phone;
```

Comment:
This is worth talking to the rest of the team about, but is it ok if we show the full phone numbers of betas to other betas? Is there a privacy concern here?